### DISCUSSION TITLE
[DISCUSSION HELP] This application requires the Windows App Runtime Version (MSIX package version > - 4000.1049.117.0)

### DISCUSSION THREAD

**Discussion by MaiderGoku** [NickvisionApps/Parabolic#760] [Q&A]
![Screenshot 2024-04-07 212947](https://github.com/NickvisionApps/Parabolic/assets/105241930/5a7faa29-df36-4dcf-8170-c2c153e9d129)
![Screenshot 2024-04-07 213856](https://github.com/NickvisionApps/Parabolic/assets/105241930/a66255a9-4e02-45ce-bc5f-13fb67595581)
![Screenshot 2024-04-07 213236](https://github.com/NickvisionApps/Parabolic/assets/105241930/867f2dae-34fd-48b2-9263-104084ff64f2)
![Screenshot 2024-04-07 213156](https://github.com/NickvisionApps/Parabolic/assets/105241930/636aab4c-a7d1-4afa-a880-2288b613246e)
![Screenshot 2024-04-07 214713](https://github.com/NickvisionApps/Parabolic/assets/105241930/ae14b883-6bbf-48a2-9838-5e4f3fc3eae7)

**Comment by nlogozzo** [comment]
#755
